CONTRIBUTING.md

```diff
@@ -172,7 +172,7 @@ docker run -t -d \
 --gpus all \
 --ipc=host \
 <docker_image_name> \
--b <debug_mode>
+-b true 
```

Comment:
should this be hardcoded true (here and below)?

Reply:
yes this is intentional for contributors who will probably want the container to persist in most cases.